eo-maven-plugin/src/test/java/org/eolang/maven/ContainsFiles.java

```diff
@@ -40,7 +40,7 @@
     "JTCOP.RuleCorrectTestName",
     "JTCOP.RuleInheritanceInTests"
 })
-final class ContainsFiles extends TypeSafeMatcher<Path> {
+public final class ContainsFiles extends TypeSafeMatcher<Path> {
```

Comment:
@Suban05 when you move the `AssembleMojoIT` to `org/eolang/maven` package - this change become unnecessary